test/files/neg/t12226.check

```diff
@@ -0,0 +1,15 @@
+t12226.scala:6: warning: Implicit resolves to enclosing class Elvis
+  implicit class Elvis[A](alt: => A) { def ?:(a: A): A = if (a ne null) a else alt } // warn
+                                                             ^
```

Comment:
I think the message should tell more precisely what's going on, is that doable? I needed `-Vprint:typer`. Also for the other universal methods.

Reply:
I'll add `try -Vprint to see what's going on`, ha. BTW in scala 2 it's not necessary to specify typer. I don't know if they'll accept the PR for dotty. I find it very convenient.